test/integration/bugs_test.go

```diff
@@ -0,0 +1,77 @@
+package integration
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"strings"
+	"testing"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var (
+	kubeClient = KubeClient()
+)
+
+// https://bugzilla.redhat.com/show_bug.cgi?id=1750665
+func TestDefaultUploadFrequency(t *testing.T) {
+	// delete any existing overriding secret
+	err := kubeClient.CoreV1().Secrets("openshift-config").Delete("support", &metav1.DeleteOptions{})
+
+	// if the secret is not found, continue, not a problem
+	if (err != nil) && (err.Error() != "secrets \"support\" not found") {
```

Comment:
`!=` operator has higher precedence that `&&`, so strictly speaking parenthesis are not needed there

(https://golang.org/ref/spec#Operators)

Reply:
you can write \`secrets \"support\" not found\` instead of escaping quote (but it depends what you prefer)

Reply:
thanks!